src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java

```diff
@@ -78,20 +106,22 @@ private static void setupLogger(Application application) {
 	/**
 	 * Launch the application.
 	 */
-	public static void main(String[] args) {
-		EventQueue.invokeLater(new Runnable() {
-			public void run() {
-				Application app = new Application("2d jobs Visio");
-				DrawerFeature.setupDrawerPlugin(app);
-				setupDefaultDrawerVisibilityManagement(app);
-
-				DriverFeature.setupDriverPlugin(app);
-				setupDrivers(app);
-				setupPresetTests(app);
-				setupLogger(app);
-
-				app.setVisibility(true);
-			}
+	public static void main(final String[] args) {
+		//pkrystian: using Runnable is old-fashioned and considered as bad practise nowadays.
+		// Replaced it with more up to date solution - java 8 lambda :)
+		EventQueue.invokeLater(() -> {
+			final Application app = new Application("2d jobs Visio");
+			DrawerFeature.setupDrawerPlugin(app);
+
+			//pkrystian: this line caused new, unnecessary window to pop out
+			//setupDefaultDrawerVisibilityManagement(app);
```

Comment:
In revision repo it is safe to delete code instead of using a comment.

Reply:
Same as before. Comment was used only to show that I'm aware of what I'm doing :)